### ISSUE TITLE
Can't use WPFLag and GPFlag together?

### ISSUE THREAD

**Issue by danrhouck** [SNL-WaterPower/CACTUS#44]
I get an error when I have both the WPFlag and GPFlag set to 1. It says "Attempting to allocate already allocated variable 'walls'". If I turn off GPFlag, it runs. This may be true if using FSFlag too, but I haven't tried. Can someone else confirm that this is the case? If so, is the idea, perhaps, that, if using a wall file, you would include your own ground plane in the wall file rather than use GPFLag?

**Comment by sakibsadman7** [comment]
Yes, I also get the same error. When using WPFlag you would need all 4 bounding walls in there.

**Comment by danrhouck** [comment]
OK. I'm actually trying to use WPFlag for some solid boundaries in the domain, not walls, but I get what you're saying. It seems like the two options share a variable in the code and can't both be on. Would be nice if you could have a ground plane while adding whatever other solid boundaries you'd like with a wall file.
